src/library/styles/components/_accordion.scss

```diff
@@ -150,7 +150,7 @@
 
     &:focus {
       @include focus-default;
-      z-index: 2;
```

Comment:
i think we should stick with 1 or 2 on this too (does this work with 1? not sure why we needed 2)